src/worldfactory.cpp

```diff
@@ -874,7 +734,52 @@ void worldfactory::draw_mod_list( WINDOW *w, int &start, int &cursor, const std:
     }
 
     wrefresh( w );
-    wrefresh(w_shift);
+    wrefresh( w_shift );
+}
+
+void worldfactory::show_active_world_mods() {
```

Comment:
You could probably use `uimenu` here instead of drawing the menu manually.

Reply:
I'm using the same function that draws the mods while creating the world (draw_mod_list()). That way I don't have to deal with ordering, naming and especially categories.